[2023-03-25 19:12:02,260] {scheduler_job.py:182} INFO - Started process (PID=202) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:02,265] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:12:02,268] {logging_mixin.py:104} INFO - [2023-03-25 19:12:02,268] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:02,319] {logging_mixin.py:104} INFO - [2023-03-25 19:12:02,296] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 13
    start_date=)
               ^
SyntaxError: invalid syntax
[2023-03-25 19:12:02,329] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:02,378] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.126 seconds
[2023-03-25 19:12:32,985] {scheduler_job.py:182} INFO - Started process (PID=205) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:32,993] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:12:33,000] {logging_mixin.py:104} INFO - [2023-03-25 19:12:32,999] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:33,067] {logging_mixin.py:104} INFO - [2023-03-25 19:12:33,061] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 15
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-03-25 19:12:33,076] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:12:33,177] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.200 seconds
[2023-03-25 19:13:03,853] {scheduler_job.py:182} INFO - Started process (PID=208) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:03,860] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:13:03,863] {logging_mixin.py:104} INFO - [2023-03-25 19:13:03,862] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:03,886] {logging_mixin.py:104} INFO - [2023-03-25 19:13:03,883] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 16
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-03-25 19:13:03,890] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:03,936] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.100 seconds
[2023-03-25 19:13:34,423] {scheduler_job.py:182} INFO - Started process (PID=211) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:34,427] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:13:34,430] {logging_mixin.py:104} INFO - [2023-03-25 19:13:34,429] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:34,450] {logging_mixin.py:104} INFO - [2023-03-25 19:13:34,446] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 16
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-03-25 19:13:34,454] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:13:34,521] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.108 seconds
[2023-03-25 19:14:05,274] {scheduler_job.py:182} INFO - Started process (PID=214) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:05,278] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:14:05,281] {logging_mixin.py:104} INFO - [2023-03-25 19:14:05,281] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:05,338] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:05,386] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.124 seconds
[2023-03-25 19:14:35,843] {scheduler_job.py:182} INFO - Started process (PID=217) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:35,847] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:14:35,851] {logging_mixin.py:104} INFO - [2023-03-25 19:14:35,851] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:35,907] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:14:35,967] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.135 seconds
[2023-03-25 19:15:06,609] {scheduler_job.py:182} INFO - Started process (PID=220) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:06,612] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:15:06,614] {logging_mixin.py:104} INFO - [2023-03-25 19:15:06,614] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:06,640] {logging_mixin.py:104} INFO - [2023-03-25 19:15:06,636] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 28
    
    ^
SyntaxError: unexpected EOF while parsing
[2023-03-25 19:15:06,646] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:06,693] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.097 seconds
[2023-03-25 19:15:37,052] {scheduler_job.py:182} INFO - Started process (PID=223) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:37,056] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:15:37,058] {logging_mixin.py:104} INFO - [2023-03-25 19:15:37,058] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:37,112] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:15:37,142] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.096 seconds
[2023-03-25 19:16:07,314] {scheduler_job.py:182} INFO - Started process (PID=226) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:07,318] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:16:07,320] {logging_mixin.py:104} INFO - [2023-03-25 19:16:07,320] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:07,344] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:07,374] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.071 seconds
[2023-03-25 19:16:37,908] {scheduler_job.py:182} INFO - Started process (PID=229) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:37,913] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:16:37,918] {logging_mixin.py:104} INFO - [2023-03-25 19:16:37,917] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:37,945] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:16:37,976] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.076 seconds
[2023-03-25 19:17:08,422] {scheduler_job.py:182} INFO - Started process (PID=232) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:08,428] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:17:08,431] {logging_mixin.py:104} INFO - [2023-03-25 19:17:08,431] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:08,460] {logging_mixin.py:104} INFO - [2023-03-25 19:17:08,456] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 31
    greet(name=)
               ^
SyntaxError: invalid syntax
[2023-03-25 19:17:08,464] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:08,526] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.110 seconds
[2023-03-25 19:17:39,534] {scheduler_job.py:182} INFO - Started process (PID=235) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:39,542] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:17:39,545] {logging_mixin.py:104} INFO - [2023-03-25 19:17:39,545] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:39,581] {logging_mixin.py:104} INFO - [2023-03-25 19:17:39,577] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 33
    greet_dag =
                ^
SyntaxError: invalid syntax
[2023-03-25 19:17:39,586] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:17:39,668] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.163 seconds
[2023-03-25 19:18:10,157] {scheduler_job.py:182} INFO - Started process (PID=238) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:10,161] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:18:10,163] {logging_mixin.py:104} INFO - [2023-03-25 19:18:10,163] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:10,233] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:10,321] {logging_mixin.py:104} INFO - [2023-03-25 19:18:10,321] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:18:10,336] {logging_mixin.py:104} INFO - [2023-03-25 19:18:10,336] {dag.py:1843} INFO - Creating ORM DAG for dag_with_taskflow_api_v01
[2023-03-25 19:18:10,364] {logging_mixin.py:104} INFO - [2023-03-25 19:18:10,363] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-24T00:00:00+00:00
[2023-03-25 19:18:10,387] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.240 seconds
[2023-03-25 19:18:41,022] {scheduler_job.py:182} INFO - Started process (PID=241) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:41,026] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:18:41,031] {logging_mixin.py:104} INFO - [2023-03-25 19:18:41,031] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:41,063] {logging_mixin.py:104} INFO - [2023-03-25 19:18:41,057] {dagbag.py:308} ERROR - Failed to import: /opt/airflow/dags/dag_with_taskflow_api.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/models/dagbag.py", line 305, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_with_taskflow_api.py", line 33
    greet_dag =  hello_world_etl()
                                 ^
IndentationError: unindent does not match any outer indentation level
[2023-03-25 19:18:41,066] {scheduler_job.py:641} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:18:41,133] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.121 seconds
[2023-03-25 19:19:11,850] {scheduler_job.py:182} INFO - Started process (PID=244) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:11,853] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:19:11,856] {logging_mixin.py:104} INFO - [2023-03-25 19:19:11,855] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:11,918] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:11,997] {logging_mixin.py:104} INFO - [2023-03-25 19:19:11,997] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:19:12,041] {logging_mixin.py:104} INFO - [2023-03-25 19:19:12,041] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-24T00:00:00+00:00
[2023-03-25 19:19:12,066] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.227 seconds
[2023-03-25 19:19:42,504] {scheduler_job.py:182} INFO - Started process (PID=247) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:42,516] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:19:42,538] {logging_mixin.py:104} INFO - [2023-03-25 19:19:42,538] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:42,617] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:19:42,750] {logging_mixin.py:104} INFO - [2023-03-25 19:19:42,750] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:19:42,802] {logging_mixin.py:104} INFO - [2023-03-25 19:19:42,801] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-24T00:00:00+00:00
[2023-03-25 19:19:42,832] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.341 seconds
[2023-03-25 19:20:13,083] {scheduler_job.py:182} INFO - Started process (PID=257) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:13,087] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:20:13,094] {logging_mixin.py:104} INFO - [2023-03-25 19:20:13,093] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:13,176] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:13,251] {logging_mixin.py:104} INFO - [2023-03-25 19:20:13,251] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:20:13,291] {logging_mixin.py:104} INFO - [2023-03-25 19:20:13,291] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:20:13,317] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.243 seconds
[2023-03-25 19:20:44,241] {scheduler_job.py:182} INFO - Started process (PID=260) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:44,245] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:20:44,248] {logging_mixin.py:104} INFO - [2023-03-25 19:20:44,248] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:44,360] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:20:44,481] {logging_mixin.py:104} INFO - [2023-03-25 19:20:44,481] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:20:44,528] {logging_mixin.py:104} INFO - [2023-03-25 19:20:44,528] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:20:44,554] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.344 seconds
[2023-03-25 19:21:15,188] {scheduler_job.py:182} INFO - Started process (PID=273) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:15,193] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:21:15,197] {logging_mixin.py:104} INFO - [2023-03-25 19:21:15,196] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:15,291] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:15,375] {logging_mixin.py:104} INFO - [2023-03-25 19:21:15,375] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:21:15,407] {logging_mixin.py:104} INFO - [2023-03-25 19:21:15,407] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:21:15,448] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.270 seconds
[2023-03-25 19:21:46,029] {scheduler_job.py:182} INFO - Started process (PID=276) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:46,035] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:21:46,039] {logging_mixin.py:104} INFO - [2023-03-25 19:21:46,038] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:46,097] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:21:46,176] {logging_mixin.py:104} INFO - [2023-03-25 19:21:46,176] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:21:46,215] {logging_mixin.py:104} INFO - [2023-03-25 19:21:46,215] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:21:46,278] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.261 seconds
[2023-03-25 19:22:16,664] {scheduler_job.py:182} INFO - Started process (PID=279) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:16,670] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:22:16,744] {logging_mixin.py:104} INFO - [2023-03-25 19:22:16,743] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:16,812] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:16,875] {logging_mixin.py:104} INFO - [2023-03-25 19:22:16,875] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:22:16,904] {logging_mixin.py:104} INFO - [2023-03-25 19:22:16,904] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:22:16,929] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.274 seconds
[2023-03-25 19:22:47,485] {scheduler_job.py:182} INFO - Started process (PID=282) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:47,489] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:22:47,491] {logging_mixin.py:104} INFO - [2023-03-25 19:22:47,491] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:47,538] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:22:47,602] {logging_mixin.py:104} INFO - [2023-03-25 19:22:47,602] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:22:47,637] {logging_mixin.py:104} INFO - [2023-03-25 19:22:47,636] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:22:47,653] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.177 seconds
[2023-03-25 19:23:18,382] {scheduler_job.py:182} INFO - Started process (PID=285) to work on /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:23:18,393] {scheduler_job.py:629} INFO - Processing file /opt/airflow/dags/dag_with_taskflow_api.py for tasks to queue
[2023-03-25 19:23:18,398] {logging_mixin.py:104} INFO - [2023-03-25 19:23:18,398] {dagbag.py:451} INFO - Filling up the DagBag from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:23:18,467] {scheduler_job.py:639} INFO - DAG(s) dict_keys(['dag_with_taskflow_api_v01']) retrieved from /opt/airflow/dags/dag_with_taskflow_api.py
[2023-03-25 19:23:18,544] {logging_mixin.py:104} INFO - [2023-03-25 19:23:18,544] {dag.py:1824} INFO - Sync 1 DAGs
[2023-03-25 19:23:18,577] {logging_mixin.py:104} INFO - [2023-03-25 19:23:18,577] {dag.py:2280} INFO - Setting next_dagrun for dag_with_taskflow_api_v01 to 2023-03-25 00:00:00+00:00
[2023-03-25 19:23:18,606] {scheduler_job.py:190} INFO - Processing /opt/airflow/dags/dag_with_taskflow_api.py took 0.261 seconds
